### ISSUE TITLE
Can our logo be used more?

### ISSUE THREAD

**Issue by gdevenyi** [CoBrALab/CobraLab.github.io#21]
Right now the logo is small and only in a single location on the page, any chance of it being more prominently displayed?

**Comment by ValiquetteVanessa** [comment]
Yes we definitely could. I will think about a nice way to show off our logo a little bit more. Thank you :)
